### PR TITLE
Document parameterized queries using SqlFormatting.

### PR DESCRIPTION
**Pull request by ddunkin**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -324,6 +324,59 @@ connector.Command(
 
 **Important note:** If the collection is empty, an `InvalidOperationException` will be thrown, since omitting the parameter entirely may not be valid (or intended) SQL.
 
+### Formatting SQL
+
+Typing parameter names in the SQL command text and parameters objects can sometimes be redundant. [`FormattableString`](https://docs.microsoft.com/en-us/dotnet/api/system.formattablestring) can be used
```
Comment (ejball) [inline docs/index.md:329]:
```suggestion
Typing parameter names in the SQL command text and parameters objects can sometimes be redundant. [String interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) can be used
```

File: docs/index.md [hunk 2]
```diff
@@ -324,6 +324,59 @@ connector.Command(
...
+    )).Execute();
+```
+
+This is equivalent to:
+
+```csharp
+var name = "First";
+var height = 6.875;
+connector.Command(
+    "insert into widgets (name, height) values (@param0, @param1);",
```
Comment (ejball) [inline docs/index.md:346]:
```suggestion
    "insert into widgets (name, height) values (@fdp0, @fdp1);",
```

Better to show the actual parameter names used?

File: docs/index.md [hunk 3]
```diff
@@ -324,6 +324,59 @@ connector.Command(
...
+    ("param0", name), ("param1", height)).Execute();
+```
+
+Note that using `FormattableString` without `Sql.Format()` is still a SQL injection vulnerability:
```
Comment (ejball) [inline docs/index.md:350]:
```suggestion
Note that using an interpolated string without `Sql.Format()` is still a SQL injection vulnerability:
```

**Review by ejball — APPROVED** [review]
Thanks!

File: docs/index.md (hunk 2 above)
Comment (ddunkin) [reply docs/index.md:346]: I wrote it that was at first, but the names are somewhat distracting, so I switched to `param`, which at least didn't leave me wondering what "fdp" means.

**Pull request merged**